topics/single-cell/tutorials/scrna_batch_correction/tutorial.md

```diff
@@ -0,0 +1,529 @@
+>        - In *"Advanced Options"*:
+>            - *"Name for dimensional reduction"*: `umap.cca`
+>
+>    > <comment-title> short description </comment-title>
+>    >
+>    > Make sure that you know what name you used for your UMAP results as we'll use this for the plots!
+>    {: .comment}
+>
+{: .hands_on}
+
+Let's see how the batch correction has changed our results. As before, we'll make one plot coloured by cluster and then another coloured by batch (`Method`). We're hoping that the batches in that second plot will be more mixed together instead of forming separate groups like they did before batch correction.
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Seurat Visualize](toolshed.g2.bx.psu.edu/repos/iuc/seurat_plot/seurat_plot/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Run Dimensional Reduction** {% icon tool %})
+>    - *"Method used"*: `Visualize Dimensional Reduction with 'DimPlot'`
+>        - *"Name of reduction to use"*: `umap.cca`
+>
+> 2. {% tool [Seurat Visualize](toolshed.g2.bx.psu.edu/repos/iuc/seurat_plot/seurat_plot/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Run Dimensional Reduction** {% icon tool %})
+>    - *"Method used"*: `Visualize Dimensional Reduction with 'DimPlot'`
+>        - *"Name of reduction to use"*: `umap.cca`
+>        - In *"Advanced Options"*:
+>            - *"Factor to group cells by"*: `Method`
+>
+{: .hands_on}
+
+![Two UMAP plots showing three large groups of cells, each made up of multiple clusters. Image A is coloured as 25 clusters. Image B shows a mix of colours across all the clusters.](../../images/scrna_batch_correction/UMAP_After_Seurat.png "UMAP after batch correction coloured by A. cluster B. Method")
+
+> <question-title></question-title>
+>
+> 1. How many clusters did we identify?
+> 2. How well mixed are the batches?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The first plot shows 25 clusters (remember that Seurat starts from cluster 0!). Although the high resolution means we still have plenty of clusters, the batch correction has reduced the number. The clusters also look less fragmented than they did before.
+> > 2. When we colour in the plot by `Method` we can see that all the colours are mixed together across all of the clusters. We don't have any clusters that are all one colour and there aren't any big patches of colour. The batch correction has successfully removed the differences between the batches so that they're no longer dominating the results.
+> >
+> {: .solution}
+>
+{: .question}
+
+</div>
+
+# Comparing the Results
+
+Before vs after batch correction
+Side by side pictures...
+Discuss what batch correction has done to the data
+
+
+
+> <comment-title></comment-title>
```

Comment:
I think this is quite important, so probably not a comment. Could show the steps via bullets, i.e. "You would then...."